include/glow/Backend/Backend.h

```diff
@@ -98,6 +98,20 @@ class Backend {
   /// \returns whether the provided \p NI is supported by the backend.
   virtual bool isOpSupported(const NodeInfo &NI) const = 0;
 
+  /// \returns whether all nodes inside \p F are supported.
+  bool checkAllNodesSupported(const Function &F) const;
+
+  /// \returns whether the provided \p F conforms to the backend-dependent graph
+  /// constraints. Giving the backend an opportunity to check that everything
+  /// conforms to its specific restrictions by overriding this function.
+  virtual bool verify(const Function &F) const;
```

Comment:
Could you add the disclaimer: `Backend verification is always a super-set of Function::verify()`.